hippunfold/workflow/rules/common.smk

```diff
@@ -24,41 +24,69 @@ def get_modality_suffix(modality):
 
 
 def get_final_spec():
-    if len(config["hemi"]) == 2:
-        specs = expand(
+    specs = expand(
+        bids(
+            root=root,
+            datatype="surf",
+            den="{density}",
+            space="{space}",
+            hemi="{hemi}",
+            label="{autotop}",
+            suffix="surfaces.spec",
+            **config["subj_wildcards"],
+        ),
+        density=config["output_density"],
+        space=ref_spaces,
+        hemi=config["hemi"],
+        autotop=config["autotop_labels"],
+        allow_missing=True,
+    )
+    return specs
+
+
+def get_final_surf():
```

Comment:
is this redundant if the surfaces are in the spec?

Reply:
I believe I added this so that we could specify mid-inner-outer surfaces in the dentate within the config file too. Without this they will never be generated since they are not in the spec. I will see with some testing though...

Reply:
I guess the better place to update it is in the expand() for the spec file then, ie if inner outer surfaces are requested then, they are put into the spec

Reply:
right, i remember now. This was a bit challenging since the config can be different for hipp and dentate surfaces.  This is likely solvable with wildcards, but I found a rule for surfaces to be a bit more intuitive since they are a major class out outputs, so its nice to have them listed explicitly. What do you think?

Reply:
would be nicer to have fewer target rules generally, but the main issue I worry about is having redundant code that gets out-of-sync, e.g. if we expand over surfaces in the cifti creation *and* the target rules -- the code so far would make it out of sync..  The `create_spec_file_dentate` rule is where you could just update what wildcards to expand over what you need (and then only have it appear once in the code).

Reply:
was just looking to undo this when i found that it had actually already made its way into the master branch:

https://github.com/khanlab/hippunfold/commit/c32bac2a3369d2e8e77137e09b3db36f983ac19e#diff-44c3321b981bf8e9ad7e5f10740de98858b868cd8a75cbbcbbe08170249441ee

I'm not quite sure what you mean about these files going out of sync though

Reply:
ok then it's something we can clean up in the v2.0 refactor -- I wasn't talking about files going out of sync, but about redundancy in the code (eg expanding over wildcards in two different places, and having to maintain those two expands to be the same..) - probably occuring more than just here, but is something we should try to avoid for easier maintenance (ie in the future when we change things, only have to change in one place, not two..)..